### PR TITLE
Disable sticky MPU in liveblogs

### PR DESCRIPTION
**Pull request by regiskuckaertz**
In liveblogs, there's no `DIV.js-content__article-body` and thus the sticky-mpu script fails during initialization. It has been decided to disable the stickiness in all cases where such an element does not exist.

Small improvement though: access to layout-related properties is now sealed in Fastdom reads.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js [hunk 1]
```diff
@@ -20,14 +18,23 @@ define([
...
+
+        element = document.querySelector(config.page.hasShowcaseMainElement ? '.media-primary' : '.content__article-body');
+        if (!element) {
+            return;
+        }
+
+        return fastdom.read(function () {
+            return element[config.page.hasShowcaseMainElement ? 'offsetHeight' : 'offsetTop'];
+        }).then(function (articleBodyOffset) {
+            this.$adSlot.parent().css('height', (articleBodyOffset + mpuHeight) + 'px');
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js]: should the `then` happen within a fastdom.write?

**Comment by johnduffell** [comment]
great! :+1: just the small q about whether the dom writing should be explicitly in a fastdom.write or whether it won't affect things

File: static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js]: yes, definitely! thanks for spotting this @johnduffell :-)

File: static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js [hunk 2]
```diff
@@ -20,14 +18,25 @@ define([
...
+        var element;
 
         if (this.$adSlot.data('name') !== 'right') {
             return;
         }
-        articleBodyOffset = config.page.hasShowcaseMainElement ? $('.media-primary').dim().height : qwery('.content__article-body')[0].offsetTop;
-        this.$adSlot.parent().css('height', (articleBodyOffset + mpuHeight) + 'px');
-        new Sticky(this.$adSlot[0], { top: this.opts.top }).init();
+
+        element = document.querySelector(config.page.hasShowcaseMainElement ? '.media-primary' : '.content__article-body');
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js]: I would move declaration of the variable here. We had many discussions about this and we are already moving to this pattern which makes more sense with ES6 anyway.

**Comment by uplne** [comment]
:+1:

**Pull request merged**